Earn: update copy and icon on the Payments splash page in Earn.

donlair opened the pull request:
This PR updates the copy and icon used on the dedicated Payments page that is shown when who has not yet connected to Stripe clicks the Payments card link in the Earn section of Calypso. Context for the changes is at pbMlHh-i6-p2.

Here are screenshots showing the changes:

Before:
<img width="1068" alt="Screen Shot 2020-06-23 at 9 24 00 PM" src="https://user-images.githubusercontent.com/35781181/85486072-eb43ce00-b597-11ea-9790-fad82684b980.png">

After:
<img width="1072" alt="Screen Shot 2020-06-23 at 3 39 25 PM" src="https://user-images.githubusercontent.com/35781181/85486088-f565cc80-b597-11ea-9ec1-7ce4d5d29f1d.png">


#### Testing instructions

* Checkout this branch (git checkout update/earn-payments-splash-page-copy-edits)
* Start calypso (yarn start)
* Navigate to the Earn page using an account with a paid plan that has not connected to Stripe (/earn)
* Click the recurring payments card to load the splash page (/earn/payments/)
* Click the CTA button and confirm that the process to connect to Stripe is started.

a8ci18n commented on client/my-sites/earn/memberships/index.jsx:
```diff
@@ -474,41 +479,62 @@ class MembershipsSection extends Component {
+						</div>
+						<div>
+							<Gridicon size={ 18 } icon="checkmark" />
+							{ this.props.translate( 'Collect payments in 135 countries.' ) }
```
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Collect payments in 135 countries' )` *ES Score: 14*

a8ci18n commented on client/my-sites/earn/memberships/index.jsx:
```diff
@@ -474,41 +479,62 @@ class MembershipsSection extends Component {
+						</div>
+						<div>
+							<Gridicon size={ 18 } icon="checkmark" />
+							{ this.props.translate( 'Easily manage subscribers.' ) }
```
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Easily manage subscribers' )` *ES Score: 11*

a8ci18n commented:
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/3924242

Thank you @donlair for including a screenshot in the description! This is really helpful for our translators.

donlair commented:
@danhauk could you take a look at the update? I added a Card component around the main content to get it closer to your updated comps. Does that also address your concerns around the heading / content alignment? I've also adjusted the spacing  and wrapping on the list.

<img width="1087" alt="Screen Shot 2020-06-24 at 3 42 37 PM" src="https://user-images.githubusercontent.com/35781181/85620311-994f8680-b631-11ea-82d2-7c5aae9d8f12.png">

a8ci18n commented on client/my-sites/earn/memberships/index.jsx:
```diff
@@ -471,46 +476,66 @@ class MembershipsSection extends Component {
 						</div>
+						<p className="memberships__onboarding-paragraph">
+							{ this.props.translate(
+								'WordPress.com Payments makes it easy to sell physical and digital goods, accept donations, charge for in-person services, build subscription newsletters, and more.'
```
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **9** times:
`translate( 'WordPress.com Payments makes it easy to sell physical and digital goods, accept donations, charge for in-person services, build subscription newsletters, and more. One-time, monthly, and yearly credit and debit card payment options are supported.' )` *ES Score: 8*

a8ci18n commented on client/my-sites/earn/memberships/index.jsx:
```diff
@@ -471,46 +476,66 @@ class MembershipsSection extends Component {
+						<p className="memberships__onboarding-paragraph">{ cta }</p>
+						<p className="memberships__onboarding-paragraph">
+							{ this.props.translate(
+								'Payments are securely processed by Stripe, a payment partner for all credit and debit card payments.'
```
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **9** times:
`translate( 'Payments are safely, securely, and seamlessly processed by Stripe, a payment partner for all credit and debit card payments. Stripe provides easy access to your money and takes just minutes to set up.' )` *ES Score: 6*

donlair commented:
Here's an updated screenshot of the current layout in this PR:
<img width="1087" alt="Screen Shot 2020-06-26 at 1 23 41 PM" src="https://user-images.githubusercontent.com/35781181/86007637-aed40000-b9e5-11ea-8a22-bd470ee2b4c0.png">

a8ci18n commented on client/my-sites/earn/memberships/index.jsx:
```diff
@@ -498,7 +445,7 @@ class MembershipsSection extends Component {
 					>
 						<NoticeAction href={ `/earn/payments-plans/${ this.props.siteSlug }` } icon="create">
-							{ this.props.translate( 'Add a payment plan' ) }
+							{ this.props.translate( 'Add a Payment Plan' ) }
```
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

a8ci18n commented:
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/3924242).

danhauk commented:
@donlair Thanks for working to bring it closer to the proposed design. I think it's looking good. I pushed up a couple commits to tweak the styles, add mobile styles, and use some typography variables we have available.

Here's some screenshots:

<img width="1064" alt="image" src="https://user-images.githubusercontent.com/448298/86282662-a8908000-bbad-11ea-84b1-e649158398ad.png">

<img width="375" alt="mobile screenshot" src="https://user-images.githubusercontent.com/448298/86282705-be05aa00-bbad-11ea-9ae2-dfd256cd6967.png">

donlair commented:
@danhauk I rebased the update and your changes look good to me. Can you mark this as reviewed? I'll deploy the changes once we get a green light that the name change updates are stable.

a8ci18n commented:
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/3924242).
